apisix/upstream.lua

```diff
@@ -327,6 +327,9 @@ function _M.set_by_route(route, api_ctx)
 
     local nodes_count = up_conf.nodes and #up_conf.nodes or 0
     if nodes_count == 0 then
+        if up_conf.parent and up_conf.parent.checker then
```

Comment:
should check the parent and checker in function `release_checker` ?

Reply:
ok sounds right. done